docs/releasing.md

```diff
@@ -23,15 +23,19 @@ The highest type before a new release determines the version update number, so i
 1. Create a branch `release-$version` in <https://github.com/owncloud/web>.
 2. Create a folder in `changelog` for the release version and date `mkdir $major.$minor.$patchVersion_YYYY-MM-DD`.
 3. Move all changelog items from the `changelog/unreleased/` folder to the `$major.$minor.$patchVersion_YYYY-MM-DD` folder.
-4. Only Web UI versions < 7.1.0: Update the version in `packages/web-integration-oc10/appinfo/info.xml`
-5. Update the version in `package.json`
-6. Commit your changes.
+4. Update the version in the following files (you can run `./dev/scripts/bump_versions.sh $version` if all get the same version):
```

Comment:
I wouldn't state all the files. In step 6 you only talk about running a script for creating and pushing all tags, so we should align with that and only instruct to use the script for bumping the versions.